google/datalab/utils/commands/_commands.py

```diff
@@ -56,15 +56,20 @@ def exit(self, status=0, message=None):
   def format_help(self):
     """Override help doc to add cell args. """
 
-    orig_help = super(CommandParser, self).format_help()
     if not self._cell_args:
-      return orig_help
+      return super(CommandParser, self).format_help()
+    else:
+      # Print the standard argparse info, the cell arg block, and then the epilog
+      epilog = self.epilog
+      self.epilog = None
```

Comment:
The epilog should be printed list. But the cell help block was being printed last. Updated comment.